pkg/plugins/helm/chart/main.go

```diff
@@ -8,14 +8,25 @@ import (
 const (
 	// CHANGELOGTEMPLATE contains helm chart changelog information
 	CHANGELOGTEMPLATE string = `
+Remark: We couldn't identify a way to automatically retrieve changelog information.
+Please use following information to take informed decision
+
 {{ if .Name }}Helm Chart: {{ .Name }}{{ end }}
 {{ if .Description }}{{ .Description }}{{ end }}
 {{ if .Home }}Project Home: {{ .Home }}{{ end }}
 {{ if .KubeVersion }}Require Kubernetes Version: {{ .KubeVersion }}{{end}}
 {{ if .Created }}Version created on the {{ .Created }}{{ end}}
-{{ if .URL }}
-Various URL:
-	{{ .URL }}
+{{ if .Sources }}
+Sources:
+{{ range $index, $source := .Sources }}
+	- {{ $source }}
+{{ end }}
+{{ end }}
+{{ if .URLs }}
+URL:
+{{ range $index, $url := .URLs }}
+	- {{ $url }}
```

Comment:
The indents are making them appear as code.

Perhaps best to leave them without indents.

So instead of 

	- https://hello.com

I would ike

- https://hello.com

Reply:
https://github.com/olblak/charts/pull/41

Reply:
Thanks for the suggestion